index.html

```diff
@@ -0,0 +1,239 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/fontawesome.min.css"
+      integrity="sha512-SgaqKKxJDQ/tAUAAXzvxZz33rmn7leYDYfBP+YoMRSENhf3zJyx3SBASt/OfeQwBHA1nxMis7mM3EV/oYT6Fdw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    
+  </head>
+  <body>
+    <section id="header-section">
+      <header id="navbar">
+        <nav>
+          <ul class="nav">
+            <li><a href="/">my logo</a></li>
+            <li class="menu-icon">
+              <button type="button" onclick="show()">
+                <img src="./Icons/Icon - Menu.svg" alt="" />
+              </button>
+            </li>
+          </ul>
+        </nav>
+
+        <nav>
+          <ul class="navbar" id="menubar">
+          <span class="close" id="close" onclick="hide()"><i class="fa fa-close"></i></span>
+            <li><a href="#headline" onclick="hide()">portfolio</a></li>
+            <li><a href="#about-myself" onclick="hide()">about</a></li>
+            <li><a href="#contact" onclick="hide()">contact</a></li>
+          </ul>
+        </nav>
+      </header>
+    </section>
+
+    <section id="headline">
+      <div class="heading-section">
+        <h1>I’m Ali Reza. Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+
+        <h3>let's connect</h3>
+
+        <ul>
+          <li>
+            <a href="/"
+              ><img src="./Icons/social/Vector-3.svg" alt="twitter logo"
+            /></a>
+          </li>
+          <li>
+            <a href="/"
+              ><img src="./Icons/social/Linkedin icon.svg" alt="linked-in logo"
+            /></a>
+          </li>
+          <li>
+            <a href="/"
+              ><img src="./Icons/social/Vector-1.svg" alt="medium logo"
+            /></a>
+          </li>
+          <li>
+            <a href="/"
+              ><img src="./Icons/social/Vector-2.svg" alt="github logo"
+            /></a>
+          </li>
+          <li>
+            <a href="/"
+              ><img src="./Icons/social/Vector.svg" alt="angle-list logo"
+            /></a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="card-works"></section>
```

Comment:
- [OPTIONAL] Your cards aren't visible. Please fix it. 
![image](https://github.com/ali-techworld/portfolio-project/assets/91486791/eb98cb8b-51d4-41bd-b0c1-eae898bdf838)